strapi/strapi
# Fix documentation plugin tests

### What does it do?

- Adds missing jest config to the documentation
- Fixes tests that were broken

### Why is it needed?

- The tests for the documentation plugin were not running

### How to test it?

From the root of the monorepo, run yarn 

```
yarn test:unit packages/plugins/documentation/server/services/__tests__
```
The tests should run and they should pass
